UNRELEASED.md

```diff
@@ -10,3 +10,6 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### Breaking changes
+
+ * Rename `--tuning-options` to `--tune-here`, see #1579
```

Comment:
I'm actually in favor of correction the docs, just because these do indeed seem to be options used for tuning, whereas I'm not at clear on what "here" means in this context.

Reply:
We currently have `--tuning=<file>` and `--tuning-options=key1=val1:key2=val2`.
I think `--tune-here` makes it clearer that tuning options for this flag go _here_, on the command line.

Or maybe these should be `--tuning-file` and `--tuning-options`?
In any case, maybe we need to be more careful in naming these things in the first place.

Reply:
What about calling the one `tuning-options-file`?

Reply:
I think we can always distinguish between the syntax `key=val` and a filename. I would prefer to introduce just one option called `--tune=<value>`, and depending on `<value>` either treat it as in-place assignments or a filename.

Reply:
> I think we can always distinguish between the syntax `key=val` and a filename. I would prefer to introduce just one option called `--tune=<value>`, and depending on `<value>` either treat it as in-place assignments or a filename.

I think the idea was that `--tune=key=val` is able to override `--tune=file`. We'd lose that ability?

Reply:
> What about calling the one `tuning-options-file`?

+1 from my side.

Reply:
> I think the idea was that `--tune=key=val` is able to override `--tune=file`. We'd lose that ability?

That is true. But the main use is to quickly define options instead of introducing a file. I doubt that we need such a complicated overriding mechanism anyways.

Reply:
I'm torn. `key=val` is a valid unix filename. How would we parse it to distinguish files from key-value pairs? Look for a valid `key`? Likely this would complicate our parsing logic for the benefit of dropping one very straight-forward flag.

Reply:
Well, `key=val` is a valid filename. I don't know. I would prefer simplicity to such a weird corner case.

Reply:
I like having a different flag for file and inline options. It's the more common alternative in my experience and precedence is easy to document.

Reply:
Yeah, let's converge on this one and merge this one :)

Reply:
It sounds like we agreed to rename `tuning` to `tuning-options-file`?

Reply:
Okay, converting this to draft until I have that renaming ready.